dashboard/app/controllers/aichat_controller.rb

```diff
@@ -49,15 +49,14 @@ def chat_completion
     # Use to_unsafe_h here to allow testing this function.
     # Safe params are primarily targeted at preventing "mass assignment vulnerability"
     # which isn't relevant here.
-    input = AichatSagemakerHelper.format_inputs_for_sagemaker_request(
+    inputs = AichatSagemakerHelper.format_inputs_for_sagemaker_request(
```

Comment:
Discussed offline - but would be good to go ahead and combine this function with `get_sagemaker_assistant_response` as well since we don't need the `inputs` for anything besides passing it to the next helper function. That way we have one clear API for getting a formatted chat response given a set of AI Chat parameters. Can do this as a follow-up as discussed.